queries/eds/highlights.scm

```diff
@@ -0,0 +1,63 @@
+["="] @punctuation.delimeter
+
+(
+ (section_name) @variable.builtin
+ (#any-of? @variable.builtin "[FileInfo]" "[DeviceInfo]" "[DummyUsage]" "[MandatoryObjects]" "[OptionalObjects]")
+)
+
+(
+ (section_name) @variable.builtin
+ (#lua-match? @variable.builtin "^%[1")
+)
+
+(
+ (section
+   (section_name) @_name
+   (#lua-match? @_name "Comments")
+   ) @comment
+ )
```

Comment:
This marks the whole section itself as a comment, not the name/body (or other nodes that are children), just want to double check that that's intended, and also the extra parenthesis pair isn't needed here and in a bunch of places below, would be nice to tidy that up

```suggestion
(section
   (section_name) @_name
   (#lua-match? @_name "Comments")) @comment
```

Reply:
yes, this is intentional that the whole section is a comment block, not just the section name.